include/freerdp/window.h

```diff
@@ -194,6 +197,10 @@ struct _WINDOW_STATE_ORDER
 	INT32 clientOffsetY;
 	UINT32 clientAreaWidth;
 	UINT32 clientAreaHeight;
+	UINT32 windowLeftResizeMargin;
+	UINT32 windowRightResizeMargin;
+	UINT32 windowTopResizeMargin;
+	UINT32 windowBottomResizeMargin;
```

Comment:
As already mentioned, please keep the old names.
we try to keep API changes to a minimum now as we're really close to 2.0

Reply:
Ok, I'll update the names. (FYI, I didn't arbitrarily rename existing fields; when I originally forked these didn't exist.)